ansible_dispatcher/cli.py

```diff
@@ -6,7 +6,7 @@
 
 import yaml
 
-from dispatcher.main import DispatcherMain
+from ansible_dispatcher.main import DispatcherMain
```

Comment:
Why not just use relative imports everywhere?

Reply:
That would be fine with me. As long as everything still worked.

Reply:
I agree, but I would prefer to just rename the existing pattern in this PR and make the change in a different PR.